apps/modernization-ui/src/design-system/item/ItemGroup.tsx

```diff
@@ -1,15 +1,20 @@
 import { ReactNode } from 'react';
 import styles from './item.module.scss';
 
-type ItemType = 'address' | 'phone' | 'name' | 'other';
+type AddressableType = 'address' | 'phone' | 'email';
+type ItemType = AddressableType | 'name' | 'other';
 
 type Props = { type?: ItemType; label?: string; children: ReactNode };
 
+const isAddressType = (type?: ItemType): type is AddressableType =>
+    type != null && ['address', 'phone', 'email'].includes(type);
+
 const ItemGroup = ({ type, label, children }: Props) => {
+    const isAddress = isAddressType(type);
     return (
         <div className={styles.itemgroup} {...(type && { 'data-item-type': type })}>
             {label && <header>{label}</header>}
-            <p>{children}</p>
+            {isAddress ? <address>{children}</address> : <p role="group">{children}</p>}
```

Comment:
question (non-blocker): Is this knowing too much information about the caller?  Should `ItemGroup` render `children` with in a `<p>` if it's a string otherwise just render `children?  This would allow the caller to specify that the child element is an address or not.

I'm not sure if that's the correct use of the group role, I think it's intended to be on the parent element containing multiple elements of some kind

Reply:
After further research <address> is not appropriate here, so I reverted to how it was before.